### PR TITLE
Fallback for shlex.split error

### PR DESCRIPTION
**Pull request by mearvind**

### CODE REVIEW

**Review by anubhms — APPROVED** [review]

File: VMBackup/main/Utils/HandlerUtil.py [hunk 1]
```diff
@@ -818,6 +818,17 @@ def command_output_from_subprocess(self , args, process_wait_time):
...
         out = str(out)
         return out
+
+    @staticmethod
+    def split(txt):
+        result = None
+        try:
+            result = shlex.split(txt)
+        except Exception as e:
+            Utils.HandlerUtil.HandlerUtility.log("Shlex.split threw exception:" + str(e))
```
Comment (vityagi) [inline VMBackup/main/Utils/HandlerUtil.py:828]: Is this working/tested ? log does not seem to be a static method.

Comment (bpramod) [reply VMBackup/main/Utils/HandlerUtil.py:828]: +1

Comment (bpramod) [inline VMBackup/main/Utils/HandlerUtil.py:828]: and also there is way to trace stacktrace. please use that way to trace exception

Comment (mearvind) [reply VMBackup/main/Utils/HandlerUtil.py]: updated the code and tested it. logging is working now

Comment (mearvind) [reply VMBackup/main/Utils/HandlerUtil.py]: added stack trace

File: VMBackup/main/Utils/HandlerUtil.py [hunk 2]
```diff
@@ -451,7 +451,7 @@ def get_total_used_size(self):
             total_used_gluster = 0
             network_fs_types = []
             for i in range(1,len(output)-1):
-                device, fstype, size, used, available, percent, mountpoint = shlex.split(output[i])
+                device, fstype, size, used, available, percent, mountpoint = output[i].split()
```
Comment (vityagi) [inline VMBackup/main/Utils/HandlerUtil.py:454]: shlex is not needed here ?

**Review by vityagi — APPROVED** [review]

**Pull request merged**